### PR TITLE
Improve Readme

### PR DESCRIPTION
**Pull request by Carreau**
Still trying to implement pep-503 unfortunately  can't get demo-data to work:

```
Traceback (most recent call last):
  File "tools/demodata.py", line 16, in <module>
    otk = st.store_user('fred', 'fredpw', 'fred@python.test')
  File "/Users/bussonniermatthias/dev/pypi-legacy/store.py", line 1563, in store_user
    if self.has_user(name):
  File "/Users/bussonniermatthias/dev/pypi-legacy/store.py", line 1550, in has_user
    safe_execute(cursor, sql, (name,))
  File "/Users/bussonniermatthias/dev/pypi-legacy/store.py", line 265, in safe_execute
    return cursor.execute(sql, safe_params)
sqlite3.OperationalError: no such table: accounts_user
```

It seam to have not been updated in a few years...

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,14 +32,33 @@ PyPI will be available in your browser at http://localhost:8000
...
+
+    driver = sqlite3
+    name = package.db
+
+
+
+Then run ``tools/demodata``    to populate the database.
 
-PyPI Requires the citext extension to be installed.
+PyPI Requires the ``citext`` extension to be installed.
```
Comment (berkerpeksag) [inline README.rst]: This probably needs to be moved to the Postgres section.

**Comment by ewdurbin** [comment]
@Carreau running a local PyPI-Legacy is indeed a bit of a mess. let me go take a look and see if some previous work in another repository still holds and i'll update here accordingly.

for now i'd caution against trying to use SQLite. i'll be circling back on this issue early AM Eastern Standard on Thursday August 4th.

File: README.rst (hunk 1 above)
Comment (Carreau) [inline README.rst]: > This probably needs to be moved to the Postgres section.

Or will move that !

**Comment by Carreau** [comment]
> @Carreau running a local PyPI-Legacy is indeed a bit of a mess. let me go take a look and see if some previous work in another repository still holds and i'll update here accordingly.
> 
> for now i'd caution against trying to use SQLite. i'll be circling back on this issue early AM Eastern Standard on Thursday August 4th.

Sure, let me know even some rough instruction and I can try to update the readme accordingly. 
I know ti is hard because of the warehouse/pypi in parallel.

**Comment by Carreau** [comment]
Also any way to  remove the immutability of https://wiki.python.org/moin/CheeseShopDev ? Which is also extremely out of date and could be improved by at least a warning that it's out of date ?

**Comment by Carreau** [comment]
Rebased.

**Comment by berkerpeksag** [comment]
> Also any way to remove the immutability of https://wiki.python.org/moin/CheeseShopDev ?

I don't know if that's the official development document these days, but see the "Editing pages" section at https://wiki.python.org/moin/ to edit a page in the wiki.

**Comment by Carreau** [comment]
> I don't know if that's the official development document these days, but see the "Editing pages" section at https://wiki.python.org/moin/ to edit a page in the wiki.

Oh great ! Done  ! Thanks !

**Comment by Carreau** [comment]
@ewdurbin did you had any chance to find some time to draft some explanations ? 

I have warehouse running locally (on Mac OS) so I suppose it is a matter of figuring out the ip/port/user/password of the docker-postgres, and setting them in config.ini ?

**Comment by ewdurbin** [comment]
@Carreau i did not unfortunately. using the warehouse setup might actually be a good path forward, though I'm not up to speed with that either.

**Comment by Carreau** [comment]
> @Carreau i did not unfortunately. using the warehouse setup might actually be a good path forward, though I'm not up to speed with that either.

No problem, thanks, I'll poke around the warehouse codebase see if I can get something.

**Comment by Carreau** [comment]
Ok, I got the database to **connect** yeah 🎊 🎉 , I got help from https://github.com/pypa/warehouse/issues/1423, still not perfect though. Readme updated to reflect the steps I had to take.

**Comment by ewdurbin** [comment]
@Carreau Thank you so much!

**Pull request merged**